blog/_posts/2023-05-30-scala-3.3.0-released.md

```diff
@@ -70,6 +70,8 @@ class TestSuite extends munit.FunSuite:
     assert(job.isSuccess)
 ```
 
```

Comment:
is it worth saying here that is can be disabled with `-source:3.2` / `import scala.language.3.2`?